internal/envoy/envoy.go

```diff
@@ -109,6 +113,7 @@ func NewServer(src config.Source, grpcPort, httpPort string) (*Server, error) {
 		httpPort:  httpPort,
 		envoyPath: envoyPath,
 	}
+	go srv.runProcessCollector()
```

Comment:
How does this get cleaned up?

Reply:
It does not. I can add code to fix this if we want, though AFAIK there's only ever one server.